### PR TITLE
Don't run docker as root

### PR DESCRIPTION
**Pull request by mathialo**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -12,4 +16,10 @@ ENV OPCUA_CERTIFICATE_DIR="/certificates"
 ENV OPCUA_OWN_CERTIFICATE_DIR="/certificates/pki/own"
 ENV OPCUA_CERTIFICATE_SUBJECT="CN=Opcua-extractor, C=NO, S=Oslo, O=Cognite, DC=docker.opcua"
 
+RUN mkdir -p /logs
+RUN chmod -R a+rw /logs
```
Comment (einarmo) [inline Dockerfile:20]: You probably have to chmod `/certificates` as well.

**Comment by mathialo** [comment]
I assume that `/logs` is the only directory the running user needs _write_ access to, it that fair?

File: Dockerfile (hunk 1 above)
Comment (mathialo) [reply Dockerfile:20]: I checked, and it should have read access, do we need write access as well?

Comment (einarmo) [reply Dockerfile:20]: Yup

**Review by einarmo — APPROVED** [review]

**Pull request merged**